.github/CODEOWNERS

```
@@ -68,6 +68,8 @@
 
 /modules/programs/broot.nix                           @aheaume
 
+/modules/programs/buf.nix                             @lucperkins
+
```

```suggestion
/modules/programs/buf.nix                             @lucperkins
/tests/modules/programs/buf                           @lucperkins

```

This is definitely common to do!

---

Ah yes! Did not catch that 😄